components/notebook-controller/README.md

```diff
@@ -35,19 +35,32 @@ All other fields will be filled in with default value if not specified.
 
 ## Environment parameters
 
-ADD_FSGROUP:  If the value is true or unset, fsGroup: 100 will be included
+ADD_FSGROUP: If the value is true or unset, fsGroup: 100 will be included
 in the pod's security context. If this value is present and set to false, it will suppress the
-automatic addition of fsGroup: 100 to the security context of the pod.  
+automatic addition of fsGroup: 100 to the security context of the pod.
 
 ## Implementation detail
 
 This part is WIP as we are still developing.
 
 Under the hood, the controller creates a StatefulSet to run the notebook instance, and a Service for it.
 
+## Contributing
+
+## Prerequisites
```

Comment:
Maybe Development Environment would be better? Also, should this be a subsection of Contributing?